code/src/cljs/sixsq/nuvla/ui/session/sign_in_views.cljs

```diff
@@ -23,6 +25,31 @@
                    ::password]))
 
 
+(defn FormTokenValidation
+  []
+  (let [tr    (subscribe [::i18n-subs/tr])
+        token (r/atom nil)]
+    (fn []
+      [comp/RightPanel
+       {:title       (str (@tr [:sign-in]) " ")
+        :title-bold  (@tr [:code-verification])
+        :FormFields  [:<>
+                      [ui/Message {:info    true
+                                   :header  (@tr [:code-verification])
+                                   :content (@tr [:two-factor-authentication-message-send])
+                                   :icon    "envelope"}]
+                      [ui/FormInput
+                       {:label         (str/capitalize (@tr [:code]))
+                        :required      true
+                        :icon          "key"
+                        :icon-position "left"
+                        :auto-focus    "on"
+                        :auto-complete "off"
+                        :value         @token
+                        :on-change     (ui-callback/input-callback #(reset! token (or (re-find #"\d+" %1) "")))}]]
+        :submit-text (str/capitalize (@tr [:validate]))
+        :submit-fn   #(dispatch [::events/validate-2fa-activation @token])}])))
+
 (defn Form
```

Comment:
consider the following flow:
 1. user logs in
 2. inserts the wrong 2FA token
 3. gets error message
 4. inserts the right one
 5. logs in
 6. logs out

after logging out, the user gets back in the login page, but the validation error message will still be there. I suspect this also applies for all other sign in error messages, and not just 2FA (due to the `(subscribe [::subs/error-message])` in the Session's RightPanel component). If you confirm this, pls open a separate ticket for this bug